dlp/google/cloud/dlp_v2/proto/dlp_pb2_grpc.py

```diff
@@ -430,52 +405,6 @@ def CancelDlpJob(self, request, context):
     context.set_details('Method not implemented!')
     raise NotImplementedError('Method not implemented!')
 
-  def CreateStoredInfoType(self, request, context):
-    """Creates a pre-built stored infoType to be used for inspection.
-    See https://cloud.google.com/dlp/docs/creating-stored-infotypes to
-    learn more.
-    """
-    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
-    context.set_details('Method not implemented!')
-    raise NotImplementedError('Method not implemented!')
-
-  def UpdateStoredInfoType(self, request, context):
-    """Updates the stored infoType by creating a new version. The existing version
-    will continue to be used until the new version is ready.
-    See https://cloud.google.com/dlp/docs/creating-stored-infotypes to
-    learn more.
-    """
-    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
-    context.set_details('Method not implemented!')
-    raise NotImplementedError('Method not implemented!')
-
-  def GetStoredInfoType(self, request, context):
-    """Gets a stored infoType.
-    See https://cloud.google.com/dlp/docs/creating-stored-infotypes to
-    learn more.
-    """
-    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
-    context.set_details('Method not implemented!')
-    raise NotImplementedError('Method not implemented!')
-
-  def ListStoredInfoTypes(self, request, context):
-    """Lists stored infoTypes.
-    See https://cloud.google.com/dlp/docs/creating-stored-infotypes to
-    learn more.
-    """
-    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
-    context.set_details('Method not implemented!')
-    raise NotImplementedError('Method not implemented!')
-
-  def DeleteStoredInfoType(self, request, context):
-    """Deletes a stored infoType.
-    See https://cloud.google.com/dlp/docs/creating-stored-infotypes to
-    learn more.
-    """
-    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
-    context.set_details('Method not implemented!')
-    raise NotImplementedError('Method not implemented!')
-
```

Comment:
Weird to have a rerun of `synthtool` remove the bits just added in this PR in f13ed406089cc8c527c440c4a4fb57a4d39699fc.  Has the `googleapis` proto / gapic config changed since 24 hours ago?

Reply:
And then that commit went away.